library/std/src/os/net/mod.rs

```diff
@@ -11,3 +11,8 @@
 )))]
 #[cfg(any(target_os = "linux", target_os = "android", doc))]
 pub(super) mod linux_ext;
+
+#[cfg(not(target_family = "windows"))]
```

Comment:
unix is not defined as !windows (there are many platforms that are neither), std/sys should have many other examples of how to correctly do the cfgs. mutex is an example